packages/cardano-services/test/entrypoints.test.ts

```diff
@@ -298,6 +313,80 @@ describe('entrypoints', () => {
         });
       });
 
+      describe('specifying a PostgreSQL-dependent services with providing service discovery args', () => {
+        let spy: jest.Mock;
+        beforeEach(async () => {
+          spy = jest.fn();
+        });
+
+        it('cli:start-server throws dns srv error and exits with code 1', (done) => {
+          expect.assertions(3);
+          proc = fork(
+            exePath('cli'),
+            [
+              'start-server',
+              '--api-url',
+              apiUrl,
+              '--postgres-srv-service-name',
+              postgresSrvServiceName,
+              '--postgres-db',
+              postgresDb,
+              '--postgres-user',
+              postgresUser,
+              '--postgres-password',
+              postgresPassword,
+              '--logger-min-severity',
+              'error',
+              '--service-discovery-backoff-timeout',
+              '1000',
+              ServiceNames.StakePool,
+              ServiceNames.NetworkInfo,
+              ServiceNames.Utxo
+            ],
+            {
+              stdio: 'pipe'
+            }
+          );
+
+          proc.stderr!.on('data', (data) => {
+            spy();
+            expect(data.toString().includes('querySrv ENOTFOUND')).toEqual(true);
+          });
+
+          proc.on('exit', (code) => {
+            expect(code).toBe(1);
+            expect(spy).toHaveBeenCalled();
+            done();
+          });
+        });
+
+        it('run throws dns srv error and exits with code 1', (done) => {
```

Comment:
```suggestion
        it('run throws DNS SRV error and exits with code 1', (done) => {
```
Also check for other cases